ci/prow-entrypoint.sh

```diff
@@ -261,13 +261,13 @@ main() {
             cosa_build
             kola_test_qemu
             ;;
-        "rhcos-92-build-test-qemu")
+        "rhcos-9-build-test-qemu")
```

Comment:
Should it not be the:
`rhcos-94-build-test-qemu`
?

Reply:
I changed it in https://github.com/openshift/release/pull/48522 to just call `rhcos-9-build-test-qemu`.

Reply:
Thanks for reviewing!

Reply:
Ah, not that I knew it but now it makes sense :)